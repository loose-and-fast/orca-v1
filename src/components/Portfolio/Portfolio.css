.Portfolio {
  height: 100%;
  background-color: #f2f2f2;
  position: relative;
}

.Portfolio h1 {
	text-align: center;
	font-family: 'PT Sans', sans-serif;
	font-weight: 525;
	font-size: 44px
}

.Portfolio h6 {
	color: #887409;
	text-align: center;
}

.Portfolio .selector {
	text-align: center;
}

.Portfolio button {
	color: white;
	background-color: #2d2c2c;
	border-color: black;
	border-radius: 12px
}

.Portfolio .btn-row {
	text-align: center;
}

.Portfolio .fa-play-circle {
	padding-right: 4px;
}