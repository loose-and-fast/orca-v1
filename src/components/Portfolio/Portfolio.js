import React from "react";
import PropTypes from "prop-types";
import { Col, Button, Row } from "react-bootstrap";
import PortfolioBlocks from "components/PortfolioBlocks/PortfolioBlocks";
import FontAwesomeIcon from '@fortawesome/react-fontawesome';
import { faPlayCircle } from '@fortawesome/fontawesome-free-solid';
import {
	camera,
	clock,
	dice,
	logo,
	lion,
	rose,
	tower,
	tracks,
	tree,
	water,
} from "assets/index.js";

import "./Portfolio.css";

const Portfolio = () => {
	console.log('camera', camera);
	return (
	<div className="Portfolio">
		<Row>
			<h6>THE SELECTED OF</h6>
			<h1>PORTFOLIO</h1>
			<div className="selector">All Items Personal Photography Portraits</div>
		</Row>
		<Row>
			<Col xs={4}>
				<PortfolioBlocks className="water" imageSrc={water} />
				<PortfolioBlocks className="tree" imageSrc={tree} />
				<PortfolioBlocks className="lion" imageSrc={lion} />
			</Col>
			<Col xs={4}>
				<PortfolioBlocks className="dice" imageSrc={dice} />
				<PortfolioBlocks className="rose" imageSrc={rose} />
				<PortfolioBlocks className="tower" imageSrc={tower} />
			</Col>
			<Col xs={4}>
				<PortfolioBlocks className="camera" imageSrc={camera} />
				<PortfolioBlocks className="tracks" imageSrc={tracks} />
				<PortfolioBlocks className="clock" imageSrc={clock} />
			</Col>			
		</Row>
		<Row className="btn-row">
			<Button onClick={()=>alert('hello')}>
				<FontAwesomeIcon className="fa-play-circle" icon={faPlayCircle} />
				Load More
			</Button>
		</Row>
	</div>);
}

Portfolio.propTypes = {};

export default Portfolio;
