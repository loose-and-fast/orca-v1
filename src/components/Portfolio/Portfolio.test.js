import React from "react";
import Portfolio from "./Portfolio";

test("Portfolio renders", () => {
  const props = {};
  const wrapper = shallow(<Portfolio {...props} />);
  expect(wrapper).toMatchSnapshot();
});
