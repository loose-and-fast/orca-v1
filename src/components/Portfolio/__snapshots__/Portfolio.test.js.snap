// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Portfolio />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Row
          bsClass="row"
          componentClass="div"
>
          <h6>
                    THE SELECTED OF
          </h6>
          <h1>
                    PORTFOLIO
          </h1>
          <div
                    className="selector"
          >
                    All Items Personal Photography Portraits
          </div>
</Row>,
        <Row
          bsClass="row"
          componentClass="div"
>
          <Col
                    bsClass="col"
                    componentClass="div"
                    xs={4}
          >
                    <PortfolioBlocks
                              className="water"
                              imageSrc="waterDrop.jpg"
                    />
                    <PortfolioBlocks
                              className="tree"
                              imageSrc="tree.jpg"
                    />
                    <PortfolioBlocks
                              className="lion"
                              imageSrc="lion.jpg"
                    />
          </Col>
          <Col
                    bsClass="col"
                    componentClass="div"
                    xs={4}
          >
                    <PortfolioBlocks
                              className="dice"
                              imageSrc="dice.jpg"
                    />
                    <PortfolioBlocks
                              className="rose"
                              imageSrc="rose.jpg"
                    />
                    <PortfolioBlocks
                              className="tower"
                              imageSrc="tower.jpg"
                    />
          </Col>
          <Col
                    bsClass="col"
                    componentClass="div"
                    xs={4}
          >
                    <PortfolioBlocks
                              className="camera"
                              imageSrc="camera.jpg"
                    />
                    <PortfolioBlocks
                              className="tracks"
                              imageSrc="tracks.jpg"
                    />
                    <PortfolioBlocks
                              className="clock"
                              imageSrc="clock.jpg"
                    />
          </Col>
</Row>,
        <Row
          bsClass="row"
          className="btn-row"
          componentClass="div"
>
          <Button
                    active={false}
                    block={false}
                    bsClass="btn"
                    bsStyle="default"
                    disabled={false}
                    onClick={[Function]}
          >
                    <FontAwesomeIcon$1
                              border={false}
                              className="fa-play-circle"
                              fixedWidth={false}
                              flip={null}
                              icon={
                                        Object {
                                                  "icon": Array [
                                                    512,
                                                    512,
                                                    Array [],
                                                    "f144",
                                                    "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                  ],
                                                  "iconName": "play-circle",
                                                  "prefix": "fas",
                                                }
                              }
                              listItem={false}
                              mask={null}
                              name=""
                              pull={null}
                              pulse={false}
                              rotation={null}
                              size={null}
                              spin={false}
                              symbol={false}
                              transform={null}
                    />
                    Load More
          </Button>
</Row>,
      ],
      "className": "Portfolio",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "row",
          "children": Array [
            <h6>
              THE SELECTED OF
</h6>,
            <h1>
              PORTFOLIO
</h1>,
            <div
              className="selector"
>
              All Items Personal Photography Portraits
</div>,
          ],
          "componentClass": "div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "THE SELECTED OF",
            },
            "ref": null,
            "rendered": "THE SELECTED OF",
            "type": "h6",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "PORTFOLIO",
            },
            "ref": null,
            "rendered": "PORTFOLIO",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "All Items Personal Photography Portraits",
              "className": "selector",
            },
            "ref": null,
            "rendered": "All Items Personal Photography Portraits",
            "type": "div",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "row",
          "children": Array [
            <Col
              bsClass="col"
              componentClass="div"
              xs={4}
>
              <PortfolioBlocks
                            className="water"
                            imageSrc="waterDrop.jpg"
              />
              <PortfolioBlocks
                            className="tree"
                            imageSrc="tree.jpg"
              />
              <PortfolioBlocks
                            className="lion"
                            imageSrc="lion.jpg"
              />
</Col>,
            <Col
              bsClass="col"
              componentClass="div"
              xs={4}
>
              <PortfolioBlocks
                            className="dice"
                            imageSrc="dice.jpg"
              />
              <PortfolioBlocks
                            className="rose"
                            imageSrc="rose.jpg"
              />
              <PortfolioBlocks
                            className="tower"
                            imageSrc="tower.jpg"
              />
</Col>,
            <Col
              bsClass="col"
              componentClass="div"
              xs={4}
>
              <PortfolioBlocks
                            className="camera"
                            imageSrc="camera.jpg"
              />
              <PortfolioBlocks
                            className="tracks"
                            imageSrc="tracks.jpg"
              />
              <PortfolioBlocks
                            className="clock"
                            imageSrc="clock.jpg"
              />
</Col>,
          ],
          "componentClass": "div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": Array [
                <PortfolioBlocks
                  className="water"
                  imageSrc="waterDrop.jpg"
/>,
                <PortfolioBlocks
                  className="tree"
                  imageSrc="tree.jpg"
/>,
                <PortfolioBlocks
                  className="lion"
                  imageSrc="lion.jpg"
/>,
              ],
              "componentClass": "div",
              "xs": 4,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "water",
                  "imageSrc": "waterDrop.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "tree",
                  "imageSrc": "tree.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "lion",
                  "imageSrc": "lion.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": Array [
                <PortfolioBlocks
                  className="dice"
                  imageSrc="dice.jpg"
/>,
                <PortfolioBlocks
                  className="rose"
                  imageSrc="rose.jpg"
/>,
                <PortfolioBlocks
                  className="tower"
                  imageSrc="tower.jpg"
/>,
              ],
              "componentClass": "div",
              "xs": 4,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "dice",
                  "imageSrc": "dice.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "rose",
                  "imageSrc": "rose.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "tower",
                  "imageSrc": "tower.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "col",
              "children": Array [
                <PortfolioBlocks
                  className="camera"
                  imageSrc="camera.jpg"
/>,
                <PortfolioBlocks
                  className="tracks"
                  imageSrc="tracks.jpg"
/>,
                <PortfolioBlocks
                  className="clock"
                  imageSrc="clock.jpg"
/>,
              ],
              "componentClass": "div",
              "xs": 4,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "camera",
                  "imageSrc": "camera.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "tracks",
                  "imageSrc": "tracks.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "className": "clock",
                  "imageSrc": "clock.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "row",
          "children": <Button
            active={false}
            block={false}
            bsClass="btn"
            bsStyle="default"
            disabled={false}
            onClick={[Function]}
>
            <FontAwesomeIcon$1
                        border={false}
                        className="fa-play-circle"
                        fixedWidth={false}
                        flip={null}
                        icon={
                                    Object {
                                                "icon": Array [
                                                  512,
                                                  512,
                                                  Array [],
                                                  "f144",
                                                  "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                ],
                                                "iconName": "play-circle",
                                                "prefix": "fas",
                                              }
                        }
                        listItem={false}
                        mask={null}
                        name=""
                        pull={null}
                        pulse={false}
                        rotation={null}
                        size={null}
                        spin={false}
                        symbol={false}
                        transform={null}
            />
            Load More
</Button>,
          "className": "btn-row",
          "componentClass": "div",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "active": false,
            "block": false,
            "bsClass": "btn",
            "bsStyle": "default",
            "children": Array [
              <FontAwesomeIcon$1
                border={false}
                className="fa-play-circle"
                fixedWidth={false}
                flip={null}
                icon={
                                Object {
                                                "icon": Array [
                                                  512,
                                                  512,
                                                  Array [],
                                                  "f144",
                                                  "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                ],
                                                "iconName": "play-circle",
                                                "prefix": "fas",
                                              }
                }
                listItem={false}
                mask={null}
                name=""
                pull={null}
                pulse={false}
                rotation={null}
                size={null}
                spin={false}
                symbol={false}
                transform={null}
/>,
              "Load More",
            ],
            "disabled": false,
            "onClick": [Function],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "fa-play-circle",
                "fixedWidth": false,
                "flip": null,
                "icon": Object {
                  "icon": Array [
                    512,
                    512,
                    Array [],
                    "f144",
                    "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                  ],
                  "iconName": "play-circle",
                  "prefix": "fas",
                },
                "listItem": false,
                "mask": null,
                "name": "",
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": null,
                "spin": false,
                "symbol": false,
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "Load More",
          ],
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Row
            bsClass="row"
            componentClass="div"
>
            <h6>
                        THE SELECTED OF
            </h6>
            <h1>
                        PORTFOLIO
            </h1>
            <div
                        className="selector"
            >
                        All Items Personal Photography Portraits
            </div>
</Row>,
          <Row
            bsClass="row"
            componentClass="div"
>
            <Col
                        bsClass="col"
                        componentClass="div"
                        xs={4}
            >
                        <PortfolioBlocks
                                    className="water"
                                    imageSrc="waterDrop.jpg"
                        />
                        <PortfolioBlocks
                                    className="tree"
                                    imageSrc="tree.jpg"
                        />
                        <PortfolioBlocks
                                    className="lion"
                                    imageSrc="lion.jpg"
                        />
            </Col>
            <Col
                        bsClass="col"
                        componentClass="div"
                        xs={4}
            >
                        <PortfolioBlocks
                                    className="dice"
                                    imageSrc="dice.jpg"
                        />
                        <PortfolioBlocks
                                    className="rose"
                                    imageSrc="rose.jpg"
                        />
                        <PortfolioBlocks
                                    className="tower"
                                    imageSrc="tower.jpg"
                        />
            </Col>
            <Col
                        bsClass="col"
                        componentClass="div"
                        xs={4}
            >
                        <PortfolioBlocks
                                    className="camera"
                                    imageSrc="camera.jpg"
                        />
                        <PortfolioBlocks
                                    className="tracks"
                                    imageSrc="tracks.jpg"
                        />
                        <PortfolioBlocks
                                    className="clock"
                                    imageSrc="clock.jpg"
                        />
            </Col>
</Row>,
          <Row
            bsClass="row"
            className="btn-row"
            componentClass="div"
>
            <Button
                        active={false}
                        block={false}
                        bsClass="btn"
                        bsStyle="default"
                        disabled={false}
                        onClick={[Function]}
            >
                        <FontAwesomeIcon$1
                                    border={false}
                                    className="fa-play-circle"
                                    fixedWidth={false}
                                    flip={null}
                                    icon={
                                                Object {
                                                            "icon": Array [
                                                              512,
                                                              512,
                                                              Array [],
                                                              "f144",
                                                              "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                            ],
                                                            "iconName": "play-circle",
                                                            "prefix": "fas",
                                                          }
                                    }
                                    listItem={false}
                                    mask={null}
                                    name=""
                                    pull={null}
                                    pulse={false}
                                    rotation={null}
                                    size={null}
                                    spin={false}
                                    symbol={false}
                                    transform={null}
                        />
                        Load More
            </Button>
</Row>,
        ],
        "className": "Portfolio",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "row",
            "children": Array [
              <h6>
                THE SELECTED OF
</h6>,
              <h1>
                PORTFOLIO
</h1>,
              <div
                className="selector"
>
                All Items Personal Photography Portraits
</div>,
            ],
            "componentClass": "div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "THE SELECTED OF",
              },
              "ref": null,
              "rendered": "THE SELECTED OF",
              "type": "h6",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "PORTFOLIO",
              },
              "ref": null,
              "rendered": "PORTFOLIO",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "All Items Personal Photography Portraits",
                "className": "selector",
              },
              "ref": null,
              "rendered": "All Items Personal Photography Portraits",
              "type": "div",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "row",
            "children": Array [
              <Col
                bsClass="col"
                componentClass="div"
                xs={4}
>
                <PortfolioBlocks
                                className="water"
                                imageSrc="waterDrop.jpg"
                />
                <PortfolioBlocks
                                className="tree"
                                imageSrc="tree.jpg"
                />
                <PortfolioBlocks
                                className="lion"
                                imageSrc="lion.jpg"
                />
</Col>,
              <Col
                bsClass="col"
                componentClass="div"
                xs={4}
>
                <PortfolioBlocks
                                className="dice"
                                imageSrc="dice.jpg"
                />
                <PortfolioBlocks
                                className="rose"
                                imageSrc="rose.jpg"
                />
                <PortfolioBlocks
                                className="tower"
                                imageSrc="tower.jpg"
                />
</Col>,
              <Col
                bsClass="col"
                componentClass="div"
                xs={4}
>
                <PortfolioBlocks
                                className="camera"
                                imageSrc="camera.jpg"
                />
                <PortfolioBlocks
                                className="tracks"
                                imageSrc="tracks.jpg"
                />
                <PortfolioBlocks
                                className="clock"
                                imageSrc="clock.jpg"
                />
</Col>,
            ],
            "componentClass": "div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": Array [
                  <PortfolioBlocks
                    className="water"
                    imageSrc="waterDrop.jpg"
/>,
                  <PortfolioBlocks
                    className="tree"
                    imageSrc="tree.jpg"
/>,
                  <PortfolioBlocks
                    className="lion"
                    imageSrc="lion.jpg"
/>,
                ],
                "componentClass": "div",
                "xs": 4,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "water",
                    "imageSrc": "waterDrop.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "tree",
                    "imageSrc": "tree.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "lion",
                    "imageSrc": "lion.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": Array [
                  <PortfolioBlocks
                    className="dice"
                    imageSrc="dice.jpg"
/>,
                  <PortfolioBlocks
                    className="rose"
                    imageSrc="rose.jpg"
/>,
                  <PortfolioBlocks
                    className="tower"
                    imageSrc="tower.jpg"
/>,
                ],
                "componentClass": "div",
                "xs": 4,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "dice",
                    "imageSrc": "dice.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "rose",
                    "imageSrc": "rose.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "tower",
                    "imageSrc": "tower.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "col",
                "children": Array [
                  <PortfolioBlocks
                    className="camera"
                    imageSrc="camera.jpg"
/>,
                  <PortfolioBlocks
                    className="tracks"
                    imageSrc="tracks.jpg"
/>,
                  <PortfolioBlocks
                    className="clock"
                    imageSrc="clock.jpg"
/>,
                ],
                "componentClass": "div",
                "xs": 4,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "camera",
                    "imageSrc": "camera.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "tracks",
                    "imageSrc": "tracks.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "className": "clock",
                    "imageSrc": "clock.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "row",
            "children": <Button
              active={false}
              block={false}
              bsClass="btn"
              bsStyle="default"
              disabled={false}
              onClick={[Function]}
>
              <FontAwesomeIcon$1
                            border={false}
                            className="fa-play-circle"
                            fixedWidth={false}
                            flip={null}
                            icon={
                                          Object {
                                                        "icon": Array [
                                                          512,
                                                          512,
                                                          Array [],
                                                          "f144",
                                                          "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                        ],
                                                        "iconName": "play-circle",
                                                        "prefix": "fas",
                                                      }
                            }
                            listItem={false}
                            mask={null}
                            name=""
                            pull={null}
                            pulse={false}
                            rotation={null}
                            size={null}
                            spin={false}
                            symbol={false}
                            transform={null}
              />
              Load More
</Button>,
            "className": "btn-row",
            "componentClass": "div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "active": false,
              "block": false,
              "bsClass": "btn",
              "bsStyle": "default",
              "children": Array [
                <FontAwesomeIcon$1
                  border={false}
                  className="fa-play-circle"
                  fixedWidth={false}
                  flip={null}
                  icon={
                                    Object {
                                                      "icon": Array [
                                                        512,
                                                        512,
                                                        Array [],
                                                        "f144",
                                                        "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                                                      ],
                                                      "iconName": "play-circle",
                                                      "prefix": "fas",
                                                    }
                  }
                  listItem={false}
                  mask={null}
                  name=""
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size={null}
                  spin={false}
                  symbol={false}
                  transform={null}
/>,
                "Load More",
              ],
              "disabled": false,
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "border": false,
                  "className": "fa-play-circle",
                  "fixedWidth": false,
                  "flip": null,
                  "icon": Object {
                    "icon": Array [
                      512,
                      512,
                      Array [],
                      "f144",
                      "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z",
                    ],
                    "iconName": "play-circle",
                    "prefix": "fas",
                  },
                  "listItem": false,
                  "mask": null,
                  "name": "",
                  "pull": null,
                  "pulse": false,
                  "rotation": null,
                  "size": null,
                  "spin": false,
                  "symbol": false,
                  "transform": null,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "Load More",
            ],
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
